samples/subsys/usb/audio/headphones_microphone/src/main.c

```diff
@@ -65,11 +65,11 @@ static const struct usb_audio_ops mic_ops = {
 
 void main(void)
 {
-	const struct device *hp_dev = device_get_binding("HEADPHONES");
+	const struct device *hp_dev = DEVICE_DT_GET_ONE(usb_audio_hp);
 	int ret;
 
 	LOG_INF("Entered %s", __func__);
-	mic_dev = device_get_binding("MICROPHONE");
+	mic_dev = DEVICE_DT_GET_ONE(usb_audio_mic);
 
 	if (!hp_dev) {
```

Comment:
Shouldn't it be replaced with `device_is_ready()` function in case of using `DEVICE_DT_GET_*` macro?